docs/designers-developers/developers/themes/theme-json.md

```diff
@@ -158,66 +166,108 @@ The list of properties that are currently exposed via this method are:
 
 ### Features
 
-This is being implemented, so it's not currently available.
+So far, this function is only enabled for the `global` section in `experimental-theme.json`.
+
+```
+{
+  "global": {
+    "features": {
+      "typography": {
+        "dropCap": false
+			}
+		}
+	}
+}
+```
+
+Then each block can decide to override how they handle block editor features during their registration process (`register_block_type` or `registerBlockType` calls) using `supports` object in `block.json` file:
+
+```
+{
+  "supports": {
+    "__experimentalFeatures": {
+      "typography": {
+        "dropCap": true
+			}
+		}
+  }
+}
+```
+
+Moving forward, we plan to integrate overrides targeting individual blocks defined inside `experimental-theme.json` file that would be applied on top of features defined by block authors in `supports` property.
+
+Finally, this is how it can be used in the block's `edit` implementation:
+
+```js
+// edit.js
+
+const Edit = ( props ) => {
+    const isDisabled = ! useEditorFeature( 'typography.dropCap', false );
```

Comment:
We may need to update this if we remove the default parameter to `useEditorFeature`:

```suggestion
    const isDisabled = ! useEditorFeature( 'typography.dropCap' );
```

Reply:
Now that I think of it, this whole part about how a block would use this property within `edit` doesn't feel like fitting this document (which is about theme.json).

Reply:
I wanted to put it somewhere. We can remove it for now.